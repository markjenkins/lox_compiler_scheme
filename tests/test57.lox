if (false)
  print "hi";
print "done";
