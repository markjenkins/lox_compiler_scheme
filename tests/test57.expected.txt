OP_FALSE
OP_JUMP_IF_FALSE 7
OP_POP
OP_CONSTANT "hi"
OP_PRINT
OP_JUMP 1
OP_POP
OP_CONSTANT "done"
OP_PRINT
OP_RETURN
